_includes/formcontent.html

```diff
@@ -0,0 +1,476 @@
+<form id="input-form" action="" method="POST" target="no-target">
+  <div class="form-group required">
+    <label class="control-label">What kind of event was this?</label>
+    <div class="radio">
+      <label>
+        <input type="radio" name="q1" value="Rust">
+        Rust
+      </label>
+    </div>
+    <div class="radio">
+      <label>
+        <input type="radio" name="q1" value="Web Compat">
+        Web Compat
+      </label>
+    </div>
+    <div class="radio">
+      <label>
+        <input type="radio" name="q1" value="WebExtension">
+        WebExtension
+      </label>
+    </div>
+    <div class="radio">
+      <label>
+        <input type="radio" name="q1" value="Host a Tech Speaker">
+        Host a Tech Speaker
+      </label>
+    </div>
+    <div class="radio">
+      <label>
+        <input type="radio" name="q1" value="Web VR">
+        Web VR
+      </label>
+    </div>
+  </div>
+  <div class="form-group required">
+    <label class="control-label">Who was the main Rep who organized this event?</label>
+    <select class="form-control" id="input-q2" name="q2">
```

Comment:
Don't bother with this for now, as I'm working on the backend to actually display some statistics on the website itself, I can just do another endpoint for this, as I already have all that information anyway.